rd-kt/rd-gen/src/main/kotlin/com/jetbrains/rd/generator/nova/kotlin/Kotlin11Generator.kt

```diff
@@ -933,10 +958,14 @@ open class Kotlin11Generator(
 
 
     protected open fun PrettyPrinter.baseClassTrait(decl: Declaration) {
+        if(isUnknown(decl)){
+            decl.implements.add(Interface("IUnknownInstance", decl.pointcut as Toplevel, emptyList()))
```

Comment:
I'm not sure we want to create new  Interface with same name for each class with such configuration. What is the purpose?

Reply:
Fixed
But in general I think, that generators can and should update interfaces list with language-specific interfaces to avoid mixing constructions like 
`p(, IPrintable)` and generation from interfaces list